tokio/src/io/driver/ready.rs

```diff
@@ -6,36 +6,32 @@ const WRITABLE: usize = 0b0_10;
 const READ_CLOSED: usize = 0b0_0100;
 const WRITE_CLOSED: usize = 0b0_1000;
 
-/// A set of readiness event kinds.
+/// Describes the readiness state of an I/O resources.
 ///
-/// `Ready` is set of operation descriptors indicating which kind of an
-/// operation is ready to be performed.
-///
-/// This struct only represents portable event kinds. Portable events are
-/// events that can be raised on any platform while guaranteeing no false
-/// positives.
+/// `Ready` tracks which operation an I/O resource is ready to perform.
 #[derive(Clone, Copy, PartialEq, PartialOrd)]
-pub(crate) struct Ready(usize);
+pub struct Ready(usize);
```

Comment:
The binary operators for `Ready` look like this:
```rust
impl<T: Into<Ready>> BitAnd<T> for Ready {}
impl<T: Into<Ready>> BitOr<T> for Ready {}
impl<T: Into<Ready>> BitOrAssign<T> for Ready {}
impl<T: Into<Ready>> Sub<T> for Ready {}
```
I would prefer to simplify this by replacing `Into<Ready>` with `Ready`, unless there's some good reason to use the `Into` trait like this?

Reply:
Applied.